src/components/UserCard.module.scss

```diff
@@ -0,0 +1,123 @@
+@import '../styles/common.scss';
+
+.user-card {
+  display: flex;
+  min-width: 296px;
+  height: 416px;
+  padding-bottom: 32px;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: flex-start;
+  flex: 1 0 0;
+  border-radius: var(--radius-md, 16px);
+  background: var(--semantic-bg-secondary, #fff);
+  box-shadow:
+    0px 4px 12px -2px rgba(24, 39, 75, 0.08),
+    0px 0px 4px 0px rgba(24, 39, 75, 0.02);
+
+  .card-top-container {
+    width: 100%;
+    height: 200px;
+    top: 0;
+    left: 0;
+    position: relative;
+
+    .user-posts-container {
+      display: flex;
+      width: 100%;
+      height: 100%;
+      justify-content: space-between;
+
+      .user-post {
+        flex: 1;
+        overflow: hidden;
+        object-fit: cover;
+
+        border-right: 1px solid var(--semantic-bg-secondary, #fff);
+
+        &:first-child {
+          border-top-left-radius: 16px; // Match card's radius
+        }
+
+        &:last-child {
+          border-top-right-radius: 16px; // Match card's radius
+          border-right: none;
+        }
+      }
+    }
+
+    .user-image {
+      display: flex;
+      align-items: flex-start;
+      gap: 8px;
+      position: absolute;
+      left: 130px;
+      bottom: -56px;
+
+      border-radius: 40px;
+      width: 80px;
+      height: 80px;
+    }
+  }
+
+  .card-bottom-container {
+    display: flex;
+    padding: 56px 24px 0px 24px;
+    flex-direction: column;
+    align-items: center;
+    gap: 24px;
+    align-self: stretch;
+
+    .user-info {
```

Comment:
this block is not really centered (probably removing `align-self: stretch;` will help)